Project/feature-u-soundboard/src/features/sounds/component/MediaControlCard.tsx

```diff
@@ -10,15 +10,20 @@ import SkipPreviousIcon from '@mui/icons-material/SkipPrevious';
 import PlayArrowIcon from '@mui/icons-material/PlayArrow';
 import SkipNextIcon from '@mui/icons-material/SkipNext';
 
-export default function MediaControlCard() {
+interface MediaControlCardProps {
+  title: string;
+  imageUrl: string;
+}
```

Comment:
Wil je hier niet een aparte file voor maken in interfaces?

Reply:
nee interface is speciafiek hiervoor